docs/government.md

```diff
@@ -212,6 +212,28 @@ pageClass: routes
 
 <Route author="sgqy" example="/go.jp/mofa" path="/go.jp/mofa"/>
 
+## 深圳市人民政府
+
+### 深圳市委组织部
+
+<Route author="zlasd" example="/shenzhengov/zzb/tzgg" path="/shenzhengov/zzb/:caty/:page?" :paramsDesc="['信息类别', '页码']">
+
+| 通知公告 | 任前公示 | 政策法规 | 工作动态 | 部门预算决算公开 | 业务表格下载 |
+| :------: | :------: | :------: | :------: | :--------------: | :----------: |
+|   tzgg   |   rqgs   |   zcfg   |   gzdt   |       xcbd       |     bgxz     |
+
+</Route>
+
+### 深圳市考试院
+
+<Route author="zlasd" example="/shenzhengov/szksy/bmxx/2" path="/shenzhengov/szksy/:caty/:page?" :paramsDesc="['信息类别', '页码']">
+
+| 通知公告 | 报名信息 | 成绩信息 | 合格标准 | 合格人员公示 | 证书发放信息 |
+| :------: | :------: | :------: | :------: | :----------: | :----------: |
+|   tzgg   |   bmxx   |   cjxx   |   hgbz   |    hgrygs    |     zsff     |
+
+</Route>
+
```

Comment:
Please put it in this section

https://github.com/DIYgod/RSSHub/blob/b36b326742e18f1f1506c3a7c59c565dfb6f1b82/docs/government.md#L61-L91